Automattic/vip-go-mu-plugins
# wp-parsely: Clear metadata for protected posts prior to 3.0.1

## Description


This is a temporary fix incorporating a change to the wp-parsely plugin released in version 3.0.1

See:
* https://github.com/Parsely/wp-parsely/pull/547
* https://github.com/Parsely/wp-parsely/blob/a525caddeac334bfecb494f0d6b170615e8f967c/CHANGELOG.md#301---2021-12-17

## Changelog Description


### Plugin Updated: wp-parsely wrapper

Incorporate hot fix from 3.0.1 for earlier versions.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change has relevant documentation additions / updates (not applicable).

## Steps to Test


1. Check out PR
2. View the source for a protected page or post without supplying the post password
3. The Parsely metadata section should not be present (ctrl+f for `<!-- BEGIN Parse.ly`, no results should be present)
4. Provide the post password and view the source again
5. The metadata section *SHOULD* now be present
6. View the source for unprotected posts, pages, and archive pages
7. The metadata section *SHOULD* be present as usual
